lib/ProductOpener/Food.pm

```diff
@@ -4241,7 +4246,7 @@ sub special_process_product($) {
 				or has_tag($product_ref, "ingredients", "sirop-de-fructose") or has_tag($product_ref, "ingredients", "saccharose")
 				or has_tag($product_ref, "ingredients", "sirop-de-fructose-glucose") or has_tag($product_ref, "ingredients", "sirop-de-glucose-fructose-de-ble-et-ou-de-mais")
 				or has_tag($product_ref, "ingredients", "sugar") or has_tag($product_ref, "ingredients", "sugars")
-
+				or has_tag($product_ref, "ingredients", "added sugar") or has_tag($product_ref, "ingredients", "added sugars")
```

Comment:
this is for ingredients, not nutrients, so "added sugars" should not be added here.